ca/certificate-authority.go

```diff
@@ -253,10 +254,10 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(csr x509.CertificateRequest
 	hostNames := make([]string, len(csr.DNSNames))
 	copy(hostNames, csr.DNSNames)
 	if len(csr.Subject.CommonName) > 0 {
-		commonName = csr.Subject.CommonName
-		hostNames = append(hostNames, csr.Subject.CommonName)
+		commonName = strings.ToLower(csr.Subject.CommonName)
```

Comment:
Okay NOW i get @jcjones's earlier comment. I thought I iterated through hostNames. My bad.

We have to leave the commonName cases, but yeah, I'll rename UniqueNames to UniqueLowerNames and go to town.

Ugh, sorry, y'all